core/src/OpenMP/Kokkos_OpenMP.hpp

```diff
@@ -127,13 +131,13 @@ class OpenMP {
   /// \brief Free any resources being consumed by the default execution space
   static void impl_finalize();
 
-  static int impl_thread_pool_size(OpenMP const& = OpenMP()) noexcept;
+  int impl_thread_pool_size() const noexcept;
+
+  inline int impl_thread_pool_size(int depth) const;
```

Comment:
Drop the inline.  It is only needed in the definition, not the declaration.
```suggestion
  int impl_thread_pool_size(int depth) const;
```